pkg/apiserver/handlers.go

```diff
@@ -495,6 +495,8 @@ func (r *APIRequestInfoResolver) GetAPIRequestInfo(req *http.Request) (APIReques
 			requestInfo.Verb = "get"
 		case "PUT":
 			requestInfo.Verb = "update"
+		case "PATCH":
+			requestInfo.Verb = "patch"
 		case "DELETE":
 			requestInfo.Verb = "delete"
 		}
```

Comment:
> should we return an error if we encounter an unknown request verb? not sure what HEAD, OPTIONS, and TRACE should map to, but right now they map to nothing

I'm not opposed to lowercasing the http verb, but I wasn't prepared to change that behavior here.